File: base64.java
import java.util.Base64;

class base64 {
    public static void main(String argv[]) {
        if (argv.length == 1) {
            final Base64.Decoder decoder = Base64.getDecoder();
            final Base64.Encoder encoder = Base64.getEncoder();
            final String text = argv[0];
            final byte[] textByte = text.getBytes();
            final String encode = encoder.encodeToString(textByte);
            final String decode = new String(decoder.decode(text));
            System.out.println("Encode: " + encode);
            System.out.println("Decode: " + decode);
        }
    }
};

File: base64.py
import base64
from sys import argv


def main():
    if len(argv) is 2:
        encode = base64.encodebytes(argv[1].encode('utf-8'))
        print('Encode: ', end='')
        print(encode.decode('utf-8'))
        decode = base64.decodebytes(argv[1].encode('utf-8'))
        print('Decode: ', end='')
        print(decode.decode('utf-8'))


if __name__ == '__main__':
    main()
